### PR TITLE
Noble Integer implementation in C, C++, Java and Python added

### LINKED ISSUE #1437: Implement noble integer in C, C++, Java and Python
@jainaman224 @rishabhgarg25699 @marwahmanbir Please assign this to me.

### PR DESCRIPTION
**Pull request by KaushalSahu**
Fixes #1437 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Noble Integer code implemented in c, cpp, java and python .

#### Languages Used:
- C
- C++
- Java
- Python

#### Files Added:
- Noble_Integer.c
- Noble_Integer.cpp
- Noble_Integer.java
- Noble_Integer.py

Thanks!

### CODE REVIEW

File: Nobel_Integer/Noble_Integer.c [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/*An integer x is said to be Noble given an array if the 
+number of integers greater than x are equal to x. 
+If there are many Noble integers, any of them is returned. 
+If there is no noble integer, then return -1.*/
+
+#include <stdio.h>
+#include <stdlib.h>
```
Comment (marwahmanbir) [inline Nobel_Integer/Noble_Integer.c]: Remove space after ``include`` in header files.

File: Nobel_Integer/Noble_Integer.cpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/*An integer x is said to be Noble given an array if the 
+number of integers greater than x are equal to x. 
+If there are many Noble integers, any of them is returned. 
+If there is no noble integer, then return -1.*/
+
+#include <bits/stdc++.h>
```
Comment (marwahmanbir) [inline Nobel_Integer/Noble_Integer.cpp]: Here also

**Comment by KaushalSahu** [comment]
@marwahmanbir please have a look before the end of the day.

**Comment by KaushalSahu** [comment]
@jainaman224 @rishabhgarg25699 please merge this else this will not be counted in score. I think @marwahmanbir is busy now.

**Comment by MastersAbh** [comment]
@KaushalSahu please squash your commits.

**Comment by KaushalSahu** [comment]
@rishabhgarg25699 @MastersAbh please review.

**Pull request merged**